.github/workflows/post_release_version_bump.yml

```diff
@@ -0,0 +1,52 @@
+name: Post-Release Version Bump
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  post_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Derive versions
+        id: versions
+        run: |
+          old_version=`grep -oP '(?<=__version__ = ").*(?=")' xir/_version.py`
```

Comment:
A problem we had on the PL- plugin side relating to version numbers was that the plugins would follow the minor version number of PL core. As a result, sometimes we wouldn't release a plugin for a particular release and then have to bump by two versions for the next release, for this reason we had to explicitly `pip install pennylane` in the GitHub action and ask for the version number. It seems XIR is independent of PL-core, but might be worth knowing about this.